test/single/example/lib/external-library-2.js

```diff
@@ -0,0 +1 @@
+console.log("external library 2 source code");
```

Comment:
Bewust gepushed? Idem andere console.log statements.

Reply:
ja das om maar iets in die scripts te staan hebben. da wordt niet uitgevoerd he, enkel de inhoud van de files wordt vergeleken met de expected, dus ik wou overal iets in staan hebben